source/includes/options-mongodrdl-commands.yaml

```diff
@@ -0,0 +1,265 @@
+
+  For example, the following command uploads the schema in the
+  ``movies.drdl`` file to the ``schemas`` database:
+
+  .. code-block:: sh
+
+     mongodrdl upload --schemaSource schemas --drdl ./movies.drdl 
+
+  The :ref:`ObjectId <document-bson-type-object-id>` of the uploaded
+  schema is returned:
+
+  .. code-block:: sh
+     :copyable: false
+
+     5d793f3f6a26a3ce66c304ea
+
+  To use the newly upload schema, restart :binary:`~bin.mongosqld` with
+  the schema database specified by the
+  :option:`--schemaSource <mongosqld --schemaSource>` option and
+  the :option:`--schemaMode <mongosqld --schemaMode>` set to ``custom``:
+
+  .. code-block:: sh
+
+     mongosqld --schemaMode custom --schemaSource <schema-db>
+
+---
+program: mongodrdl
+name: download
+directive: commandoption
+description: |
+  Downloads a schema specified by its name or
+  :ref:`ObjectId <document-bson-type-object-id>` and prints it to the
+  console.
+
+  ``download`` accepts the following options:
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 20 80
+
+     * - Name
+       - Description
+
+     * - ``--schemaSource``
+       - *Required.* Specifies the database where the schema information
+         is stored.
+
+     * - ``--name``
+       - The name of the schema. To learn more about naming schemas,
+         see the :commandoption:`name-schema <mongodrdl --name-schema>`
+         command.
+
+         *Required* if ``--schema`` is not specified.
+
+     * - ``--schema``
+       - The :ref:`ObjectId <document-bson-type-object-id>` of the
+         schema.
+
+         *Required* if ``--name`` is not specified.
+  
+  For example, the following command prints the ``movies`` schema to the
+  console:
+
+  .. code-block:: sh
+
+     mongodrdl download --schemaSource schemas --name movies
+  
+  To store the schema in a ``.drdl`` file, cat the output to the desired
+  ``.drdl`` file:
+
+  .. code-block:: sh
+
+     mongodrdl download --schemaSource schemas --name movies > ./movies.drdl
+
+---
+program: mongodrdl
+name: delete
+directive: commandoption
+description: |
+  Deletes a schema specified by its name or
+  :ref:`ObjectId <document-bson-type-object-id>`. 
+
+  ``delete`` accepts the following options:
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 20 80
+
+     * - Name
+       - Description
+
+     * - ``--schemaSource``
+       - *Required.* Specifies the database where the schema information
+         is stored.
+
+     * - ``--name``
+       - The name of the schema. To learn more about naming schemas,
+         see the :commandoption:`name-schema <mongodrdl --name-schema>`
+         command.
+
+         *Required* if ``--schema`` is not specified.
+
+     * - ``--schema``
+       - The :ref:`ObjectId <document-bson-type-object-id>` of the
+         schema.
+
+         *Required* if ``--name`` is not specified.
+
+  For example, the following command deletes the
+  ``5d7941dc6a26a3d0fc397284`` schema in the ``schemas`` database:
```

Comment:
I'd say "deletes the schema with id ``5d7941dc6a26a3d0fc397284`` in the ``schemas`` database:"